src/frontends/pytorch/include/openvino/frontend/pytorch/node_context.hpp

```diff
@@ -130,22 +128,22 @@ class NodeContext : public frontend::NodeContext {
         }
     }
 
-    Output<Node> get_tensor_from_model_or_create_input(size_t index);
+    Output<Node> get_tensor_from_model_or_create_input(size_t index) const;
     Output<Node> get_input_from_visible_context(size_t index) const;
-    std::shared_ptr<ov::Model> convert_subgraph(size_t index);
+    std::shared_ptr<ov::Model> convert_subgraph(size_t index) const;
 
 private:
     std::shared_ptr<TorchDecoder> m_decoder;
-    std::set<size_t> m_mutated_tensors;
-    TensorMap* m_tensor_map;
     const TensorMap& m_ext_tensor_map;
-    ParameterVector* m_external_parameters;
-    TranslateSession* m_translate_session;
+    TensorMap* m_tensor_map = nullptr;
+    ParameterVector* m_external_parameters = nullptr;
+    std::set<size_t>* m_mutated_tensors = nullptr;
+    TranslateSession* m_translate_session = nullptr;
```

Comment:
can we use smart pointers please? It will be more safe.

Reply:
done